content/self-host/rustdesk-server-pro/smtp/microsoft-365/_index.es.md

```
@@ -22,7 +22,7 @@ Para la configuracion SMTP general, consulte [SMTP](../).
 | OAuth2 Client secret | El `Value` del secreto creado en `Certificates & secrets` |
 
 Esta captura muestra donde se introducen estos valores en RustDesk:
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/microsoft-365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)